tasks/Containerfile

```diff
@@ -27,14 +29,17 @@ RUN dnf -y update && \
         libvirt-client \
         libvirt-python3 \
         libXt \
+        mtools \
         nc \
         net-tools \
         nodejs-devel \
         npm \
+        ostree \
```

Comment:
What do you need this for? I'm rather sceptical that this will work in an unprivileged container. You can run `toolbox create --image quay.io/cockpit/tasks -c cockpit` to test it.

Reply:
it is based on dependencies inside podman image what was used before: https://github.com/loadtheaccumulator/fleet-iso-util/

Reply:
But https://github.com/loadtheaccumulator/fleet-iso-util/blob/main/fleetkick.sh doesn't use ostree anywhere?